### DISCUSSION TITLE
配置问题

### DISCUSSION THREAD

**Discussion by htl** [Qexo/Qexo#429] [General]
你好作者，我觉得你的这个后台貌似设计的不够灵活，在我这，如果可以，你可以增加自定义目录配置的功能，而不是限定友链等页面仅仅只是你指定的目录，有可能一些使用者很早以前就搭建好了博客，已经指定好了目录，如果能够配置左侧选项卡的目录就再好不过

**Comment by am-abudu** [comment]
我不明白你的意思。需要明确的是，这些所有的功能和你使用的主体、博客、配置无任何关系
